src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const AMOUNT_$_PER_DAY = 40;
+  const DISCOUNT_7_DAYS = 50;
+  const DISCOUNT_3_DAYS = 20;
```

Comment:
Don't use symbols like `$`, `%` in variable names. Same recommendation for numbers here (use words):
```suggestion
  const AMOUNT_PER_DAY = 40;
  const LONG_TERM_DISCOUNT = 50;
  const SHORT_TERM_DISCOUNT = 20;
```